cli/internal/run/run.go

```diff
@@ -487,14 +490,6 @@ func parseRunArgs(args []string, output cli.Ui) (*RunOptions, error) {
 		return nil, errors.Errorf("At least one task must be specified.")
 	}
 
-	cwd, err := os.Getwd()
```

Comment:
Maybe it would result in less churn to pass this in? I think we have it already on the `Config` object, right?

Reply:
Feedback addressed.